data/35e/wizards_of_the_coast/supplement/book_of_vile_darkness/bv_abilitycategories.lst

```diff
@@ -22,7 +22,7 @@ ABILITYCATEGORY:Thrall of Orcus Class Feature			VISIBLE:QUALIFY	EDITABLE:YES	EDI
 ABILITYCATEGORY:Ur-Priest Class Feature				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO						CATEGORY:Special Ability	TYPE:Ur-Priest Class Feature				PLURAL:Ur-Priest Class Features				DISPLAYLOCATION:Class Features
 ABILITYCATEGORY:Vermin Lord Class Feature				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO						CATEGORY:Special Ability	TYPE:Vermin Lord Class Feature			PLURAL:Vermin Lord Class Features				DISPLAYLOCATION:Class Features
 ABILITYCATEGORY:Warrior of Darkness Class Feature		VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO						CATEGORY:Special Ability	TYPE:Warrior of Darkness Class Feature		PLURAL:Warrior of Darkness Class Features			DISPLAYLOCATION:Class Features
-
+ABILITYCATEGORY:Thrall of Orcus Choice					VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:YES					CATEGORY:Special Ability	TYPE:OrcusChoice									PLURAL:Thrall of Orcus Choices		DISPLAYLOCATION:Class Features
```

Comment:
New ability category to handle per-level choice of Bonus Feat or Spells for Thrall of orcus